runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java

```diff
@@ -44,28 +44,39 @@ default PrayerFlickLocation prayerFlickLocation()
 
 	@ConfigItem(
 		position = 1,
+		keyName = "prayerFlickAlwaysOn",
+		name = "Pray flick helper always on",
```

Comment:
Change this to "Never hide prayer flick helper"

Reply:
done